README.md

```diff
@@ -0,0 +1,80 @@
++ [How Does It Work?](https://github.com/bitdust-io/docs/blob/master/principle.md)
++ [Installation](https://github.com/bitdust-io/docs/blob/master/install.md)
++ [Commands](https://github.com/bitdust-io/docs/blob/master/commands.md)
++ [Settings](https://github.com/bitdust-io/docs/blob/master/settings.md)
++ [Glossary](https://github.com/bitdust-io/docs/blob/master/glossary.md)
+
+
+## Working Principles
+
++ [User Identification](https://github.com/bitdust-io/docs/blob/master/identities.md)
++ [Distributed Storage](https://github.com/bitdust-io/docs/blob/master/storage.md)
++ [Data Security](https://github.com/bitdust-io/docs/blob/master/security.md)
++ [Automatic Rebuilding](https://github.com/bitdust-io/docs/blob/master/rebuilding.md)
++ [Distributed Hash-Table](https://github.com/bitdust-io/docs/blob/master/dht.md)
++ [Network services](https://github.com/bitdust-io/docs/blob/master/services.md)
++ [Transport protocols](https://github.com/bitdust-io/docs/blob/master/transports.md)
++ [Crypto-contracts](https://github.com/bitdust-io/docs/blob/master/crypto_contracts.md)
+
+
+## Development
+
++ [Roadmap](https://github.com/bitdust-io/docs/blob/master/roadmap.md)
++ [Finite State Machines](https://github.com/bitdust-io/docs/blob/master/automats.md)
++ [Technologies](https://github.com/bitdust-io/docs/blob/master/technologies.md)
++ [API Interface](https://github.com/bitdust-io/docs/blob/master/api.md)
++ [Known bugs](https://github.com/bitdust-io/docs/blob/master/bugs.md)
+
+
+## Deploying
+
+You can just open "index.html" file in the root folder of your local "www" repository and fully automated and independent copy of the web site will be rendered in your web browser. The web site do not require database to be avaialble on your host and only serve static HTML pages at the moment.
+
+On the live machine serving https://bitdust.io resource "www" repository is cloned and direcly loaded into WWW folder maintained by Apache2 web server. When your "Pull Request" will be merged we will execute "./update" caommand to pull changes on the live machine.
+
+
+## Contributing
+
+This repository contains .html files generated from [BitDust documentation](https://github.com/bitdust-io/docs) sources built in Markdown format. You can re-generate all of the BitDust Wiki pages this way:
+
+    git clone https://github.com/bitdust-io/docs.git bitdust.docs
+    git clone https://github.com/bitdust-io/www.git bitdust.www
+    cd bitdust.www
+    ./build
+
+If you want to modify any page on the web site, create a fork of both repositories on GitHub and start making changes in corresponding .md file of "docs" repo. Then save the file and run "./build" in "www" repo - a new .html file will be generated with fresh changes you did in the .md file.
+
+Commit and push your changes to your fork repository and start a "Pull Request", BitDust team is doing public code reviews and we will be in touch with you to go further.
+
```

Comment:
You can make a fork of the repository and clone it.
Then you can add "main" repository as "upstream" to be able to keep your local files in sync with remote files.:

```
cd bitdust.www/
git remote add upstream https://github.com/bitdust-io/www.git
```